### DISCUSSION TITLE
Task mentioned in Go Time podcast

### DISCUSSION THREAD

**Discussion by marco-m** [go-task/task#1106] [General]
Around minute 38 of https://changelog.com/gotime/271
Used by https://github.com/wailsapp/wails 

> The traditional method of providing web interfaces to Go programs is via a built-in web server. Wails offers a different approach: it provides the ability to wrap both Go code and a web frontend into a single binary. Tools are provided to make this easy for you by handling project creation, compilation and bundling. 

💪

**Comment by andreynering** [comment]
Thanks for letting us know 🙂
